src/acl/class.test.ts

```diff
@@ -35,6 +37,8 @@ import {
   unstable_AclDataset,
   unstable_WithAcl,
 } from "../interfaces";
+import { Quad } from "rdf-js";
+import { foaf } from "../constants";
```

Comment:
Could you create a ticket about extracting constants from tests throughout the codebase? Now we'll have this test which uses one centrally-defined constant, and the rest defined inline, which I can imagine can confuse future contributors.

Reply:
Created here: https://inrupt.atlassian.net/browse/SDK-1176 (although all this was already done in my previous PR!).